scalding-core/src/main/scala/com/twitter/scalding/Execution.scala

```diff
@@ -852,11 +870,22 @@ object Execution {
       { case (conf, _, w, cec) => w.getForced(conf, t)(cec) })
 
   def toIterable[T](t: TypedPipe[T]): Execution[Iterable[T]] =
+    from(UUID.randomUUID()).flatMap { uuid =>
```

Comment:
let's add a test that: `Execution.toIterable(p) == Execution.toIterable(p)`. I think we fail that currently.

Reply:
since we include uuid into `ToWrite.ToIterable` they can not be the same. is it a problem?

Reply:
okey, got what you mean.

Reply:
added tests and fixed equality.